Description: Disable building of some internal tools that no longer link
 because internal symbols are not exported.
Author: Joachim Bauch <bauch@struktur.de>
--- a/Makefile.am
+++ b/Makefile.am
@@ -8,10 +8,6 @@
 SUBDIRS+=dec265
 endif
 
-SUBDIRS+=enc265
-SUBDIRS+=tools
-SUBDIRS+=acceleration-speed
-
 if ENABLE_SHERLOCK265
 SUBDIRS+=sherlock265
 endif
--- a/dec265/Makefile.am
+++ b/dec265/Makefile.am
@@ -1,5 +1,5 @@
 
-bin_PROGRAMS = dec265 hdrcopy
+bin_PROGRAMS = dec265
 
 AM_CPPFLAGS = -I../libde265
 
@@ -9,12 +9,6 @@
 dec265_LDADD = ../libde265/libde265.la -lstdc++
 dec265_SOURCES = dec265.cc
 
-hdrcopy_DEPENDENCIES = ../libde265/libde265.la
-hdrcopy_CXXFLAGS =
-hdrcopy_LDFLAGS =
-hdrcopy_LDADD = ../libde265/libde265.la -lstdc++
-hdrcopy_SOURCES = hdrcopy.cc
-
 if HAVE_VIDEOGFX
   dec265_CXXFLAGS += $(VIDEOGFX_CFLAGS)
   dec265_LDFLAGS += $(VIDEOGFX_LIBS)

